Samples/Desktop/D3D12Raytracing/readme.md

```diff
@@ -23,6 +23,13 @@ This sample demonstrates how to implement procedural geometry using intersection
 
 ![D3D12 Raytracing Procedural Geometry GUI](src/D3D12RaytracingProceduralGeometry/Screenshot_small.png)
 
+
+## 4. [Library Subobject Sample](src/D3D12LibrarySubobjects/readme.md)
+This sample modifies the simple lighting sample, and illustrates how to use library subobjects.
```

Comment:
Please follow other samples with exact name and full path reference, i.e.
[D3D12 Raytracing Simple Lighting sample](../D3D12RaytracingSimpleLighting/readme.md).

Reply:
GitHub embedded the link in the comment. Trying again "[D3D12 Raytracing Simple Lighting sample](../D3D12RaytracingSimpleLighting/readme.md)."

Reply:
Again embedded. Take a look at .md in ProceduralGeometry sample for reference.